e2e-app/src/app/position/scroll/scroll.component.html

```diff
@@ -0,0 +1,21 @@
+<h3>Position with scroll tests</h3>
+
+<div>
+  <button class="btn btn-outline-secondary ml-1" id="container-null" (click)="container = null">Append to widget</button>
+  <button class="btn btn-outline-secondary ml-1" id="container-body" (click)="container = 'body'">Append to body</button>
+</div>
+<div id="outer-container" class="p-3" style="height: 400px; overflow: scroll">
+  <div id="inner-container" class="position-relative d-inline-block p-3" style="width: 600px; height: 200px; overflow: scroll;">
```

Comment:
just a nitpick, but is there a reason for setting width to `600px` for these tests?